Pod/Classes/JNKeychain.m

```diff
@@ -36,7 +36,7 @@ + (NSMutableDictionary *)getKeychainQuery:(NSString *)key forAccessGroup:(NSStri
         [keychainQuery setObject:[self getFullAppleIdentifier:group] forKey:(__bridge id)kSecAttrAccessGroup];
     }
     
-    return [keychainQuery mutableCopy];
```

Comment:
mutableCopy / copy methods always return a retained object which the caller is responsible for releasing.
When we’re creating an object using explicit allocation then we’re responsible for it and within the autorelease pool block it is automatically released when a callee doesn’t longer need an object.